src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -10,42 +10,50 @@
 helps['login'] = """
 type: command
 short-summary: Log in to Azure.
-long-summary: >-
-    By default, this command logs in with a user account. CLI will try to launch a web browser to log in interactively.
-    If a web browser is not available, CLI will fall back to device code login.
-
-    To login with a service principal, specify --service-principal.
 examples:
-    - name: Log in interactively.
-      text: az login
-    - name: Log in with user name and password. This doesn't work with Microsoft accounts or accounts that have two-factor authentication enabled. Use -p=secret if the first character of the password is '-'.
-      text: az login -u johndoe@contoso.com -p VerySecret
-    - name: Log in with a service principal using client secret. Use -p=secret if the first character of the password is '-'.
-      text: az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p VerySecret --tenant contoso.onmicrosoft.com
-    - name: Log in with a service principal using client certificate.
-      text: az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p ~/mycertfile.pem --tenant contoso.onmicrosoft.com
-    - name: Log in using a VM's system-assigned managed identity.
-      text: az login --identity
-    - name: Log in using a VM's user-assigned managed identity. Client or object ids of the service identity also work.
-      text: az login --identity -u /subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID
+- name: Log in interactively.
+  text: az login
+- name: Log in with user name and password. This doesn't work with Microsoft accounts or accounts that have two-factor authentication enabled.
+  text: az login --user johndoe@contoso.com --password VerySecret
+- name: Log in with a password containing a `-` as the first character.
+  text: az login --user johndoe@contoso.com --password=-myPasswordThatStartsWithAdash
+- name: Log in with a service principal using a client secret.
+  text: az login --service-principal --user http://azure-cli-2016-08-05-14-31-15 --password VerySecret --tenant contoso.onmicrosoft.com
+- name: Log in with a service principal using a client certificate.
+  text: aaz login --service-principal --user http://azure-cli-2016-08-05-14-31-15 --password ~/MyCertfile.pem --tenant contoso.onmicrosoft.com
+- name: Log in using a VM's system assigned identity.
```

Comment:
```suggestion
- name: Log in using default managed assigned identity.
```

Not only VM has managed identity. 

Also, if no system assigned identity is configured, but a user assigned identity is, this command uses the user assigned identity as the default one.